consensus/src/aggregation/engine.rs

```diff
@@ -75,11 +80,9 @@ pub struct Engine<
     TSu: ThresholdSupervisor<
         Index = Epoch,
         PublicKey = P,
+        Polynomial = Vec<V::Public>,
         Share = group::Share,
-        Identity = poly::Public<V>,
```

Comment:
It is incorrect to use polynomial as Identity because it changes each epoch (whereas Identity is meant to be stable).